### ISSUE TITLE
FR: Option for the date format of highlight extractions

### ISSUE THREAD

**Issue by ichnilatis-gr** [koreader/koreader#12418]
**Does your feature request involve difficulty completing a task? Please describe.**
While in the Bookmark details the date is in international ISO 8601 format (YYYY-MM-DD), in the Html extractions of the highlights the date is in US format (MM-DD-YYYY). Could there be a possibility to choose the date format for the Html extractions or, better, the date be in the international format, as it is in the Bookmark details?

Thank you!

**Comment by hius07** [comment]
It's here
https://github.com/koreader/koreader/blob/f3005906d4236fa234a789b8e62df4ea7ed1dcea/plugins/exporter.koplugin/template/note.tpl#L70
Replace `"%x"` with `"%Y-%m-%d %H:%M:%S"`. No need to restart.

**Comment by ichnilatis-gr** [comment]
Thank you very much! But do I have to make this change after every update?

**Comment by hius07** [comment]
Yes, other users might be happy with the current template, nobody has complained.

**Comment by ichnilatis-gr** [comment]
Well, let it be... since I'm the strange one... :-)
